auth0-training/labs-node-create-an-application-ilt
# Update web app to use Auth Code flow & update codetour to reflect changes

## Testing instructions:

Run through the CodeTours and ensure that line highlights and code insertions are as expected, with special attention to the Add AuthN CodeTour.

Note: Currently it seems like the Auth0 Labs Extension is not populating the CLIENT_SECRET, so you will have to add it manually into .env:

`CLIENT_SECRET = <value>`
